examples/github-action.yml

```diff
@@ -34,5 +34,6 @@ jobs:
         run: |
           git config user.name github-actions
           git config user.email github-actions@github.com
+          git add .
           git commit -am "Compiled firmware"
```

Comment:
No need for ´commit -am´ as we now use ´add .´, replace with ´commit -m´ and this PR is good to go.